### PR TITLE
Fix all E203, E741 and W504 flake8 errors

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

**Review by jmchilton — APPROVED** [review]

**Comment by jmchilton** [comment]
Clicking approve because I think this is a big improvements. But I'll give some time before merging because some people prefer to line up their white spaces in sprawling dicts and such. I think that was mostly Carl that really cared but I've seen others do it.

File: lib/galaxy/config/script.py [hunk 1]
```diff
@@ -154,7 +154,8 @@ def _handle_galaxy_yml(args, config_dir, data_dir):
 
     galaxy_config_template = []
     with open(GALAXY_CONFIG_TEMPLATE_FILE) as fh:
-        for line in [l.rstrip('\n') for l in fh.readlines()]:
+        for line in fh.readlines():
```
Comment (bgruening) [inline lib/galaxy/config/script.py]: Can we remove readlines here? Maybe a task a another PR?

**Review by bgruening — APPROVED** [review]

File: lib/galaxy/config/script.py (hunk 1 above)
Comment (nsoranzo) [reply lib/galaxy/config/script.py]: Done, thanks!

**Review by mvdbeek — APPROVED** [review]

**Comment by mvdbeek** [comment]
There's a conflict here, can you resolve @nsoranzo ?

**Comment by nsoranzo** [comment]
> There's a conflict here, can you resolve @nsoranzo ?

Sure, rebased.

**Comment by mvdbeek** [comment]
Thank you @nsoranzo!

**Pull request merged**